include/vcpkg/base/message-args.inc.h

```diff
@@ -29,6 +29,10 @@ DECLARE_MSG_ARG(exit_code, "127")
 DECLARE_MSG_ARG(expected_version, "1.3.8")
 DECLARE_MSG_ARG(extension, ".exe")
 DECLARE_MSG_ARG(feature, "avisynthplus")
+DECLARE_MSG_ARG(
+    git_tree_sha,
+    "7cfad47ae9f68b183983090afd6337cd60fd4949 (see "
+    "https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftree-ishatree-ishalsotreeish )")
```

Comment:
I don't see how this makes a difference for localization compared to `commit_sha`. I think adding the url is excessive.

However, I'm fine with adding `git_tree_sha` in general as an named argument.

Reply:
I think it can be helpful for localizers if they know what we are talking about. Supplying the URI certainly doesn't *hurt* anything.